pom.xml

```diff
@@ -60,6 +60,9 @@
     <sat.version>0.5.0</sat.version>
     <slf4j.version>1.7.21</slf4j.version>
     <xtext.version>2.14.0</xtext.version>
+    <jetty.version>9.4.12.v20180830</jetty.version>
```

Comment:
This Jetty version is used with Karaf 4.2.2 but not with Karaf 4.2.1 (still defined on line 59). See also the Karaf 4.2.2 upgrade [changes](https://github.com/openhab/openhab-core/pull/458/files).

If the goal is to make a working openhab-distro using the versions defined in this POM I think it's best to update Karaf to 4.2.2.

Reply:
FTR, this version is required [here](https://github.com/openhab/openhab-core/blob/master/features/karaf/openhab-core/src/main/feature/feature.xml#L582-L594).

I just see that a Jetty version is defined in https://github.com/openhab/openhab-core/blob/master/features/karaf/openhab-tp/pom.xml#L17, so maybe it is best to move this one to https://github.com/openhab/openhab-core/blob/master/features/karaf/openhab-core/pom.xml then.

It would make the overall setup much simpler though, if we could simply say that openhab-core requires Karaf 4.2.2. @maggu2810 what's your take on this?

Reply:
I've moved the properties to the openhab-core feature for now - would suggest to merge it as is then (as it fixes a concrete bug that prevents the distro from being built), while I am open to discuss a general restructuring of features & versions for the future.